buildstockbatch/schemas/v0.3.yaml

```diff
@@ -28,6 +28,7 @@ gcp-spec:
   # notifications_email: regex('^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$', name='email', required=True)
   gcs: include('gcs-spec', required=True)
   use_spot: bool(required=False)
+  job_environment: include('gcp-job-environment', required=False)
```

Comment:
you don't think `use_spot` should be part of `job_environment` now?

Reply:
I thought about that and was on the fence. I left it for consistency with the AWS schema... but I think you're right that it would be better inside of `job_environment`, so I'll switch to that.